tests/Unit/inc/Engine/CriticalPath/RESTWPPost/generate.php

```diff
@@ -199,6 +199,15 @@ function( $post_id ) use ( $expected ) {
 			}
 		}
 
+		if ( $expected['success'] && 'cpcss_generation_successful' === $expected['code'] ) {
+			Functions\expect( 'rocket_clean_files' )
+				->atMost()
+				->times( 1 )
```

Comment:
Is there a case where the function would not be called?

Reply:
You are correct. In that situation exactly 1 time it should be called.
Updated the code to reflect this.